File: src/backend/src/main/java/br/com/grupopipa/gestaointegrada/financeiro/titulo/TituloSetorDTO.java
package br.com.grupopipa.gestaointegrada.financeiro.titulo;

import java.math.BigDecimal;
import java.util.UUID;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class TituloSetorDTO {
  private UUID setorId;
  private String setorNome;
  private BigDecimal percentualRateio;
}

File: src/frontend/src/app/components/financeiro/titulo/model/titulo-setor-dto.ts
export class TituloSetorDTO {
  setorId: string;
  setorNome?: string;
  percentualRateio: number;

  constructor(setorId: string, percentualRateio: number, setorNome?: string) {
    this.setorId = setorId;
    this.percentualRateio = percentualRateio;
    this.setorNome = setorNome;
  }
}
